addons/web/models/ir_http.py

```diff
@@ -121,7 +122,7 @@ def session_info(self):
                     mods, request.session.context['lang']
                 ) if session_uid else None,
             },
-            "currencies": self.sudo().get_currencies(),
+            "currencies": self.env['res.currency'].sudo().get_all_currencies(),
```

Comment:
I know that you keep the old behavior but it doesn't sound correct to have the sudo **outside** the ormcache method. It means that the content of the cache depends of the first user that fill it.